api/python/provisioner/commands/__init__.py

```diff
@@ -431,6 +438,34 @@ def run(self, *args, targets=ALL_MINIONS, dry_run=False, **kwargs):
 
         return self._run(params, targets)
 
+@attr.s(auto_attribs=True)
+class PillarDump(CommandParserFillerMixin):
+    input_type: Type[inputs.PillarKeysList] = inputs.PillarKeysList
+    _run_args_type = RunArgsPillarDump
+
+    @classmethod
+    def from_spec(
+        cls, input_type: str = 'PillarKeysList'
+    ):
+        return cls(input_type=getattr(inputs, input_type))
+
+
+    def run(
+        self, *args, targets: str = ALL_MINIONS, local: bool = False,
+        **kwargs
+    ):
+        res = PillarGet.run(self)
+        os.makedirs(f"{PRVSNR_CORTX_CONFIG_DIR}", exist_ok=True)
```

Comment:
please use [pathlib.mkdir](https://docs.python.org/3.6/library/pathlib.html#pathlib.Path.mkdir) here instead, please note that PRVSNR_CORTX_CONFIG_DIR is already a Path object

Reply:
Yes, Addressed now.